modules/nf-core/phispy/main.nf

```diff
@@ -30,7 +30,12 @@ process PHISPY {
         -o $prefix \\
         $gbk
 
-    mv ${prefix}/${prefix}_prophage_coordinates.tsv ${prefix}.tsv
+    mv ${prefix}/${prefix}_prophage_coordinates.tsv ${prefix}/${prefix}.tsv
+
+    filename=$gbk
+    gbk_extension="\${filename#*.}"
+    mv ${prefix}/${prefix}_${gbk} ${prefix}/${prefix}.\${gbk_extension}
```

Comment:
I'm thinking if it's really viable to move all files to the top-level directory. Because the gbk output, for example, is basically a duplicate of the input gbk but with added prophage regions. So if I move it up and make it the name equal to the prefix, there's a chance it overwrites the input file. Not that it's necessarily an issue since it's just a symlink, but I just just wanted to point this out. What do you think, @nvnieuwk?

Reply:
You can add a check for filename collisions to avoid that (a lot of modules do this actually). The docs have a nice explanation on this ([here](https://nf-co.re/docs/contributing/modules#naming-conventions) part 6)

Reply:
Oh awesome. I hadn't noticed that. Thanks.

Reply:
Done [3122ddc](https://github.com/nf-core/modules/pull/3453/commits/3122ddc5cad39c0965c5a6600a6ff99f23de6692)